src/sdc11073/schema_resolver.py

```diff
@@ -57,3 +63,14 @@ def resolve(self, url, id, context):  # pylint: disable=unused-argument, redefin
         with open(path, 'rb') as my_file:
             xml_text = my_file.read()
         return self.resolve_string(xml_text, context, base_url=path)
+
+    def _get_schema_file_path(self, url: str) -> Union[str, None]:
+        """
+
+        :param url: url of the schema location
+        :return: str or None
+        """
+        for entry in self.namespaces:
```

Comment:
can be replaced by `next` method
`return next((entry for entry in self.namespaces if entry.schema_location_url == url), None)`

Reply:
well, almost ;)
return next((entry.local_schema_file for entry in self.namespaces if entry.schema_location_url == url), None)